libraries-3/pom.xml

```diff
@@ -8,8 +8,9 @@
 
     <parent>
         <groupId>com.baeldung</groupId>
-        <artifactId>parent-modules</artifactId>
-        <version>1.0.0-SNAPSHOT</version>
+        <artifactId>parent-boot-2</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+        <relativePath>../parent-boot-2</relativePath>
```

Comment:
Why have you changed the parent?

Reply:
Since we have a web-server, I decided to add the Spring Boot parent which also has the previous parent as its parent.